tests/integration/common/transaction_utils.rs

```diff
@@ -186,6 +202,32 @@ pub fn to_internal_invoke_v0_tx(tx: &InvokeTransactionV0) -> InternalTransaction
     };
 }
 
+/// Convert a InvokeTransactionV0 to a SNOS InternalTransaction
+pub fn to_internal_invoke_v1_tx(tx: &InvokeTransactionV1) -> InternalTransaction {
+    let max_fee = tx.max_fee.0.into();
+    let signature = Some(tx.signature.0.iter().map(|x| to_felt252(x)).collect());
+    let entry_point_selector = Some(to_felt252(&selector_from_name("__execute__").0));
+    let calldata = Some(tx.calldata.0.iter().map(|x| to_felt252(x.into())).collect());
+    let contract_address = to_felt252(tx.sender_address.0.key());
+    let nonce = felt_api2vm(tx.nonce.0);
+    let hash_value = tx_hash_invoke_v1(contract_address, calldata.clone().unwrap(), max_fee, nonce);
```

Comment:
I think we should implement this converter as `impl TryFrom<InvokeTransactionV1> for InternalTransaction.` May be left for another PR.